docs/typography.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Unstyled } from "@storybook/blocks"
+
+<Meta title="essentials/Typography" />
+
+# Typography
+
+The typography comprises five predefined font sizes. The base font size of the
+app is 14px when used with the default browser settings.
```

Comment:
Do we agree that `14px` is the baseline? I remember discussing 14-15-16px with Mario. I'm fine with 14px, but just saying.

Reply:
![CleanShot 2024-09-09 at 11 59 58](https://github.com/user-attachments/assets/9e2eb131-5d3a-4ca9-82e3-ef09d11dd589)